### PR TITLE
first draft of create a pipeline

### PR DESCRIPTION
**Pull request by TyDunn**

### CODE REVIEW

**Review by adrianbr — APPROVED** [review]

File: docs/website/docs/walkthroughs/create-a-pipeline.md [hunk 1]
```diff
@@ -2,4 +2,142 @@
...
+Google BigQuery from scratch. The same steps can be repeated for any source and destination of your 
+choice--use `dlt init <source> <destination>` and then build the pipeline for that API instead.
+
+Please make sure you have [installed `dlt`](./installation.mdx) before getting started here.
+
+## 1. Initialize project
+
+Create a new empty directory for your `dlt` project by running
+```
+mkdir twitter-bigquery
```
Comment (rudolfix) [inline docs/website/docs/walkthroughs/create-a-pipeline.md:17]: should we suggest to create virtual env? or just to refer to quickstart?

**Review by rudolfix — COMMENTED** [review]
OK this looks good. When I have twitter pipeline reviewed and new init script I will go through it again and see if code examples are ok

File: docs/website/docs/walkthroughs/create-a-pipeline.md [hunk 2]
```diff
@@ -2,4 +2,142 @@
...
+## 3. Add Twitter API credentials
+
+You will need to [sign up for the Twitter API](https://developer.twitter.com/en/docs/platform-overview)
+and create a project to grab your Bearer Token for the Twitter API.
+
+Copy the value of the bearer token into `.dlt/secrets.toml`
+```
+[sources]
+
+twitter_bearer_token = '<bearer token value>'
```
Comment (rudolfix) [inline docs/website/docs/walkthroughs/create-a-pipeline.md:44]: Here I'd mention that this value must correspond to the name of the argument here:
`twitter_bearer_token=dlt.secrets.value`
in `def twitter_resource(api_url=dlt.config.value, twitter_bearer_token=dlt.secrets.value):`

and if they want to change the name of the secret they also must change the name of the argument (and vice versa)

**Review by rudolfix — COMMENTED** [review]
I have a general comment on how we do it here and what is in the init template in the repo:

we show people how to build twitter pipeline and our init template contain a lot of twitter specific code.

the people that do not want to build twitter thing are in a difficult position because of that. They need to remove twitter-specific code

should we start with a generic template and the first thing we should tell them is to name their sources and resources? (this can be done by `dlt init` command to some degree)

also @adrianbr should demonstrate that as a first step. IMO

once they named their source and first resource they can proceed with the other stuff - printing out the secret values and writing the part where authorization header is created

File: docs/website/docs/walkthroughs/create-a-pipeline.md (hunk 1 above)
Comment (TyDunn) [reply docs/website/docs/walkthroughs/create-a-pipeline.md:17]: We tell them to create a virtual env in installation: "Please make sure you have [installed `dlt`](./installation.mdx) before getting started here."

**Comment by TyDunn** [comment]
Merging for now... believe I have addressed all comments, but I would double check and fix in a new PR

**Pull request merged**